base/scripts/docker-compose-reports-redirect.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  frontend:
+    labels:
```

Comment:
Could you add a comment to explain the purpose of those labels? and outline that this is some kind of non-standard requirement